src/tags/app-switch/app-switch.style.module.scss

```
@@ -1,7 +1,25 @@
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    border: 0;
+    overflow: hidden;
+    clip-path: inset(50%);
+  }
+
+  // The input carries the focus but cannot show it, so the icon stands in.
+  &:has(> input:focus-visible) {
+    outline: 2px solid currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)